dangerfile.ts

```diff
@@ -49,6 +49,18 @@ been edited. Are you sure you don’t want to also relocate them to the new [fil
 View the [frontend file org ADR](https://github.com/transcom/mymove/blob/main/docs/adr/0048-frontend-file-org.md) for more information`);
   }
 
+  const deprecatedDevSeedFiles = danger.git.fileMatch(
+    'pkg/testdatagen/scenario/devseed.go',
+    'pkg/testdatagen/scenario/subscenarios.go',
+  );
```

Comment:
I wonder if we should also be checking for changes in `cmd/generate-test-data/main.go` as well here @StuTruss? That file will eventually be included in the deletion of the `devseed` scenarios as we won't be needing it to generate test data in that way again.

```suggestion
  const deprecatedDevSeedFiles = danger.git.fileMatch(
    'pkg/testdatagen/scenario/devseed.go',
    'pkg/testdatagen/scenario/subscenarios.go',
    'cmd/generate-test-data/main.go',
  );
```